app/src/androidTest/java/com/example/sharingang/payment/FakePaymentProvider.kt

```diff
@@ -9,7 +9,7 @@ class FakePaymentProvider : PaymentProvider {
         // Don't have to initialize anything
     }
 
-    override suspend fun requestPayment(item: Item): Boolean =
+    override suspend fun requestPayment(item: Item, quantity: Int): Boolean =
         if (item.id == null || item.price < 0.01) false else paymentStatus.status
```

Comment:
Should you also check here that quantity is more than 0 to be sure?

Reply:
Can you do that check here anyway?

Reply:
Same